tests/app/db.py

```diff
@@ -567,13 +567,22 @@ def create_api_key(service, key_type=KEY_TYPE_NORMAL, key_name=None):
     return api_key
 
 
-def create_inbound_number(number, provider='mmg', active=True, service_id=None):
+def create_inbound_number(
+    number,
+    provider='mmg',
+    active=True,
+    service_id=None,
+    url_endpoint: str = None,
```

Comment:
Either all have type annotations or none. Very odd to only specify for only one parameter.

Reply:
removed